packages/base-driver/lib/basedriver/core.ts

```diff
@@ -258,13 +265,35 @@ export class DriverCore<const C extends Constraints, Settings extends StringReco
    * @param name - name of feature/command
    */
   isFeatureEnabled(name: string): boolean {
+    // automationName comparison is case-insensitive,
+    // while feature name is case-sensitive
+    const currentAutomationName = _.toLower(this.opts.automationName);
+
+    const parseFullName = (fullName: string) => {
+      const separatorPos = fullName.indexOf(FEATURE_NAME_SEPARATOR);
+      if (separatorPos < 0) {
+        // This should not happen as we preprocess corresponding server arguments in advance
+        throw new Error(
+          `The full feature name must include both the driver name/wildcard and the feature ` +
+          `name split by a colon, got '${fullName}' instead`
+        );
+      }
+      return [
+        _.toLower(fullName.substring(0, separatorPos)),
+        fullName.substring(separatorPos + 1)
+      ];
+    };
+    const parseFullNames = (fullNames: string[]) => fullNames.map(parseFullName);
+    const matches = ([automationName, featureName]: [string, string]) =>
+      [currentAutomationName, ALL_DRIVERS_MATCH].includes(automationName) && featureName === name;
+
```

Comment:
It looks like we still need https://github.com/appium/appium/pull/20793/files#diff-bb56f5d13605c22db7ab1c490736e963c7c1449a8056c6c509fbc470fcbd2dfcR472-R494 into the base driver since environment which doesn't have this latest appium version such as `appium@2.5.4` installs this driver but the appium package doesn't have `*` workaround by this PR, so this behaves a breaking change for such environment.

I think our intention is for future preparation so still we won't require this prefix kind. Or we needed to bump this base driver's major version to not affect such old Appium packages